Shared/VisualCRedistributableVersionFunctions.ps1

```diff
@@ -14,10 +15,15 @@ function Get-VisualCRedistributableInstalledVersion {
         $softwareList = New-Object 'System.Collections.Generic.List[object]'
     }
     process {
-        $installedSoftware = Invoke-ScriptBlockHandler -ComputerName $ComputerName `
-            -ScriptBlock { Get-ItemProperty HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\* } `
-            -ScriptBlockDescription "Querying for software" `
-            -CatchActionFunction $CatchActionFunction
+
+        if ($PSSenderInfo) {
```

Comment:
For all the `Shared` functions, see if we can't remove this logic and just treat it like it is always running locally.